sections/installation.js

```diff
@@ -29,7 +29,7 @@ export default [
       $ npm install sqlite3
       $ npm install mysql
       $ npm install mysql2
-      $ npm install oracle
+      $ npm install oracledb
```

Comment:
I would recommend `npm install oracledb@3.1.2` instead because latest one is not yet supported: https://github.com/knex/knex/issues/3396